### PR TITLE
Use nunjuck template engine to render data in sponsor.yml file

### PR DESCRIPTION
**Pull request by railsstudent**
#49

### CODE REVIEW

**Review by yookoala — COMMENTED** [review]
Possible typo in `assets/page/sponsor/partial.jinja`.

Could `sponsor.logs.img` be `sponsor.logos.img` or something else? (Or should it be here? Seems duplicate with the `p.logo` section.)

File: assets/pages/sponsor/partial.jinja [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<div class="sponsor-type">
+  <div class="row">
+    <div class="col m3">
+      {{sponsor.logs.img}}
```
Comment (yookoala) [inline assets/pages/sponsor/partial.jinja]: Possible typo in assets/page/sponsor/partial.jinja.

Could sponsor.logs.img be sponsor.logos.img or something else? (Or should it be here? Seems duplicate with the p.logo section.)

File: assets/data/sponsor.yml [hunk 2]
```diff
@@ -1,43 +1,44 @@
-sponsors:
- -  name: "Gold Sponsor"
+sponsor_types:
+  - name:     "Gold"
```
Comment (yookoala) [inline assets/data/sponsor.yml]: To synchronize with changes in #53, this should be "Gold Sponsor" instead of "Gold".

**Review by yookoala — CHANGES_REQUESTED** [review]
Please rebase the commit from #53. (merge conflict now)

File: assets/pages/sponsor/partial.jinja (hunk 1 above)
Comment (railsstudent) [reply assets/pages/sponsor/partial.jinja]: Typing error. removed now.

File: assets/data/sponsor.yml (hunk 2 above)
Comment (railsstudent) [reply assets/data/sponsor.yml]: Amended.

**Comment by railsstudent** [comment]
are you going to merge it?
@tonyhhyip

**Comment by tonyhhyip** [comment]
@railsstudent Need to wait for you to rebase it

**Comment by railsstudent** [comment]
I thought I rebase it.

**Comment by tonyhhyip** [comment]
Need to wait for @yookoala review

**Comment by railsstudent** [comment]
i think i just rebased master to this branch.

**Comment by tonyhhyip** [comment]
@railsstudent You did rebase but we need to wait for @yookoala to review

**Review by yookoala — APPROVED** [review]

**Comment by yookoala** [comment]
Thanks Connie for the work :-)

**Pull request merged**